### ISSUE TITLE
history_file directive ignored in ini

### ISSUE THREAD

**Issue by thorstenkampe** [dbcli/litecli#62]
The `history_file` directive in the ini file is ignored. Litecli always writes to the default location.

**Comment by amjith** [comment]
There is no `history_file` directive in the ini file. It always writes to ~/.config/litecli/history. 

I've tried to reduce this number of knobs of configure. Can you tell me what use case you're trying to address with this?

**Comment by thorstenkampe** [comment]
You are correct. `Mycli` and `Litcli` don't support that directive while `pgcli` and `mssql-cli`. I'm using a single ini for all four dbcli applications so I got misled.

There were two use cases:
1. a single history file for all four dbcli shells I use. Since there are a few but significant syntax differences in the respective SQL dialects that turned out to be not useful.
2. a "portable" location for my history so I can take it with me an don't leave traces in the host (portable meaning on a USB thumb drive). That's less of an issue as it used to be.

The main issue is probably that I would wish that the dbcli shells would be more similar. From what I can see there are already two kinds of dbcli apps: the `\n` ones (`pgcli` and `mssql-cli`) and the `\f` ones (`litecli` and `mycli`). One uses `named queries`, the other one `favorite queries`, etc.

Please feel free to close this issue.
